.buildkite/scripts/pipelines/security_solution_quality_gate/security_solution_cypress/mki_security_solution_cypress.sh

```diff
@@ -16,6 +16,9 @@ export JOB=kibana-security-solution-chrome
 
 buildkite-agent meta-data set "${BUILDKITE_JOB_ID}_is_test_execution_step" "true"
 
+mkdir .ftr
+retry 5 5 vault kv get -format=json -field=data secret/kibana-issues/dev/security-quality-gate/role-users > .ftr/role_users.json
```

Comment:
We recently implemented a change where we're accessing `vault` through a variable path. I think for this to be future-proof to some degree, we should also care for that branching in here. However, the utility function that we use cannot be parameterized with extra args in its current state.

If this is urgent, we can go with this.
If you have some time to chisel the edges, I'll follow up with the details, so you can add it to this PR.

Reply:
@dkirchan can you please take a look at @delanni comment and estimate how long is going to take to make the changes? Thanks!

Reply:
@delanni as discussed private, lets unblock this PR and I will add these changes as part of my PR that I am preparing: https://github.com/elastic/kibana/pull/173005